Directory.Packages.props

```diff
@@ -34,8 +34,8 @@
     <!-- this was the previous version used -->
     <PackageVersion Include="System.Text.Json" Version="5.0.2" />
     <GlobalPackageReference Include="PolySharp" Version="1.14.1" />
-    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
+    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>
```

Comment:
GlobalPackageReference shouldn't need this

Reply:
cool, thought I'd check with you. Makes sense